### DISCUSSION TITLE
请问历史版本去哪里找，我想找基于ubuntu18的最后一个版本

### DISCUSSION THREAD

**Discussion by yssylq** [ophub/amlogic-s9xxx-armbian#1415] [Other]
Device Information | 设备信息

SOC: s905x3
H96 Max X3
Armbian Version | 系统版本

Kernel Version: Armbian_23.02.2 & 5.15.106
Release: jammy & bullseye

请问历史版本去哪里找，我想找基于ubuntu18的最后一个版本

**Comment by ophub** [comment]
f大的度盘里有，他的帖子里有3个盘的地址和口令。

https://www.right.com.cn/forum/thread-4076037-1-1.html
